Sources/HttpPipelineTestSupport/HttpPipelineTestSupport.swift

```diff
@@ -5,6 +5,12 @@ import Prelude
 import SnapshotTesting
 import XCTest
 
+extension Application {
+  var isOther: Bool {
+    return !["javascript", "json", "xml", "x-www-form-url-encoded"].contains(self.rawValue)
```

Comment:
This is kinda a hack just for this file. Wonder if we can tackle the logic that depends on it in a different way.